src/main/java/com/lmax/disruptor/IgnoreExceptionHandler.java

```diff
@@ -15,43 +15,32 @@
  */
 package com.lmax.disruptor;
 
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import java.lang.System.Logger.Level;
+import java.lang.System.Logger;
 
 /**
- * Convenience implementation of an exception handler that using standard JDK logging to log
- * the exception as {@link Level}.INFO
+ * Convenience implementation of an exception handler that uses the standard JDK logging
+ * of {@link System.Logger} to log the exception as {@link Level}.INFO
  */
 public final class IgnoreExceptionHandler implements ExceptionHandler<Object>
 {
-    private static final Logger LOGGER = Logger.getLogger(IgnoreExceptionHandler.class.getName());
-    private final Logger logger;
-
-    public IgnoreExceptionHandler()
-    {
-        this.logger = LOGGER;
-    }
-
-    public IgnoreExceptionHandler(final Logger logger)
-    {
-        this.logger = logger;
-    }
+    private static final Logger LOGGER = System.getLogger(IgnoreExceptionHandler.class.getName());
 
     @Override
     public void handleEventException(final Throwable ex, final long sequence, final Object event)
     {
-        logger.log(Level.INFO, "Exception processing: " + sequence + " " + event, ex);
+        LOGGER.log(Level.INFO, "Exception processing: " + sequence + " " + event, ex);
```

Comment:
Given the unknown complexity of `event::toString` I wonder if, on the `IgnoreExceptionHandler` at least, we should use the supplier variant of `LOGGER::log`?
Then if using the ignore handler, and you have a log level higher above `INFO` it won't spend time building that string?

Reply:
Good idea. I can't think of any serious downsides to using the supplier, so I'll be consistent and use that in both handlers.